lesson3/hometask_argirovatina.rb

```diff
@@ -0,0 +1,31 @@
+module MyEach
+
+  def my_each
+    self.to_a.flatten if self.is_a?(Hash)
```

Comment:
Why do you _flat_ it?
Is it expected to you: `{a: 1, b: 2}.to_a.flatten # => [:a, 1, :b, 2]` ?